### PR TITLE
Html for public IP resource

### PR DESCRIPTION
**Pull request by shylasrinivas**

### CODE REVIEW

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -20,22 +20,22 @@ resources.
...
-token from the Cloud Service Provider by providing the org scoped API token. 
-The Terraform provider client uses Cloud Service Provider CSP API 
-to exchange this org scoped API token for user access token. 
+token from the Cloud Service Provider by providing the org scoped API token.
+The Terraform provider client uses Cloud Service Provider CSP API
+to exchange this org scoped API token for user access token.
 
 There are also a number of other parameters that can be set to tune how the
-provider connects to the VMC Console API. 
+provider connects to the VMC Console API.
```
Comment (rchadalawada) [inline website/docs/index.html.markdown]: What are those other parameters? If are not documenting those, we should either remove this line or provide a link to more info on those params

Comment (shylasrinivas) [reply website/docs/index.html.markdown]: @saloneerege can you help with this comment? It was present in the markdown prior to my change.

Comment (shylasrinivas) [reply website/docs/index.html.markdown]: Deleted this line based on my discussion with @saloneerege

**Review by rchadalawada — APPROVED** [review]

File: website/docs/r/public_ip.html.markdown [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  display_name = var.public_ip_displayname
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported for vmc_sddc resource:
+
+* `org_id` - (Required) Organization identifier.
```
Comment (saloneerege) [inline website/docs/r/public_ip.html.markdown]: I think we should only mention the arguments that are applicable to the resource public ip . We can add a reference to the sddc resource index.html page for the other resources.

**Review by saloneerege — APPROVED** [review]

**Pull request merged**